python/test/t_ProjectionStrategy_std.expout

```diff
@@ -0,0 +1,164 @@
++ Compute flood model by regression
```

Comment:
idem

Reply:
Fixed at 9ad6b67d1a0db5a1fbaa46ba37c20c3873c52757

Reply:
Idem. I think that removing the expected output file was an error after all.